engine/docker/docker.go

```diff
@@ -38,20 +38,22 @@ import (
 )
 
 const (
-	imageMaxRetries           = 3
-	imageRetrySleepDuration   = 50
-	networkMaxRetries         = 3
-	networkRetrySleepDuration = 50
-	harnessHTTPProxy          = "HARNESS_HTTP_PROXY"
-	harnessHTTPSProxy         = "HARNESS_HTTPS_PROXY"
-	harnessNoProxy            = "HARNESS_NO_PROXY"
-	dockerServiceDir          = "/etc/systemd/system/docker.service.d"
-	httpProxyConfFilePath     = dockerServiceDir + "/http-proxy.conf"
-	directoryPermission       = 0700
-	filePermission            = 0600
-	windowsOS                 = "windows"
-	removing                  = "removing"
-	running                   = "running"
+	imageMaxRetries                  = 3
+	imageRetrySleepDuration          = 50
+	startContainerRetries            = 10
+	startContainerRetrySleepDuration = 5
+	networkMaxRetries                = 3
+	networkRetrySleepDuration        = 50
```

Comment:
Assuming these are in seconds, sounds very long.

Reply:
assuming you are taking about networkRetrySleepDuration, this is in ms, been there since some time, not a change of this pr